modules/cvv/samples/cvv_demo.cpp

```diff
@@ -78,6 +72,7 @@ main(int argc, char** argv)
     printf("Setting resolution to %dx%d\n", resolution->width, resolution->height);
     capture.set(CV_CAP_PROP_FRAME_WIDTH, resolution->width);
     capture.set(CV_CAP_PROP_FRAME_HEIGHT, resolution->height);
+    delete resolution;
```

Comment:
@Sahloul Could you please replace resolution definition to be `cv::Size` structure instead of pointer as @alalek suggests.

Reply:
It is optional again (checked here `if (resolution)`)

Reply:
Sorry, missed that.